src/JustEat.StatsD/Buffered/Tags/StatsDTagsFormatter.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Collections.Generic;
+
+namespace JustEat.StatsD.Buffered.Tags
+{
+    internal sealed class StatsDTagsFormatter : IStatsDTagsFormatter
+    {
+        private readonly IStatsDTagsFormatter _inner;
+
+        public StatsDTagsFormatter(TagsStyle tagsStyle)
+        {
+            _inner = tagsStyle switch
```

Comment:
As noted earlier in the review, let's expose this without an enum and require the user to configure a specific `IStatsDTagsFormatter` if they want to use tags.

Reply:
Done.